packages/raystack/components/announcement-bar/announcement-bar.module.css

```
@@ -41,7 +41,19 @@
+  align-items: center;
   gap: var(--rs-space-1);
   cursor: pointer;
+  background: transparent;
+  border: 0;
+  padding: 0;
+  font: inherit;
+  color: inherit;
+}
+
+.action-btn:focus-visible {
+  outline: 2px solid currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)